vscode-extension/PUBLISHING.md

```
@@ -2,13 +2,15 @@
 
 ## Prerequisites
 
+> **⚠️ Deprecation Notice:** The Visual Studio Marketplace listing is deprecated. Please use the [Open VSX Registry](https://open-vsx.org/extension/EffortlessMetrics/perl-lsp-rs) for publishing.
+
 1. **Node.js and npm** installed
 2. **Visual Studio Code** installed
 3. **vsce** (Visual Studio Code Extension manager) and **ovsx** (Open VSX CLI) installed:
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)